docs/querying/sql-functions.md

```diff
@@ -729,11 +790,30 @@ Rounds down a timestamp by a given time unit.
 
 ## FLOOR (numeric)
 
-`FLOOR(<NUMERIC>)`
+Rounds down to the largest integer value less than or equal to the numeric expression.
```

Comment:
This description is confusing. What does it round down? Maybe keep the original?

> Calculates the largest integer less than or equal to the numeric expression.

Reply:
I use "Round down ..." to be consistent with the definition used for `FLOOR (date and time)`. That said, it appears documentation from microsoft and w2schools use a similar definition to the original, so I will change it for consistency. I will do the same for CEIL as well